tracing-subscriber/src/fmt/format/pretty.rs

```diff
@@ -241,12 +241,14 @@ impl<'a> PrettyVisitor<'a> {
         Self { style, ..self }
     }
 
-    fn maybe_pad(&mut self) {
-        if self.is_empty {
+    fn write_padded(&mut self, value: &dyn fmt::Debug) {
```

Comment:
take it or leave it: this might be _slightly_ more efficient if we used `impl Debug` so that it gets monomorphized?
```suggestion
    fn write_padded(&mut self, value: &impl fmt::Debug) {
```